evalml/pipelines/components/ensemble/stacked_ensemble_base.py

```diff
@@ -93,10 +93,6 @@ def fit(self, X, y=None):
         Returns:
             self
         """
-        if isinstance(X, pd.DataFrame):
-            X = X.to_numpy()
-        if isinstance(y, pd.Series):
-            y = y.to_numpy()
```

Comment:
Yupperino! The issue with the stacked ensemble was `Exception during automl search: could not convert string to float: 'CUC'`; that's because of these lines to `.to_numpy()` which tried to convert the input data to float. Without these lines, we can pass the data to fit, where the input pipelines have components to handle the string data :)

Reply:
Aha! Makes sense. So we pass pandas now. Thanks!